src/cloudflare/internal/d1-api.ts

```diff
@@ -6,7 +6,7 @@ interface Fetcher {
   fetch: typeof fetch
 }
 
-type D1Response = {
+interface D1Response {
```

Comment:
unrelated changes in this file?

Reply:
eslint formated the file by mistake, fixed